apps/arweave/src/ar_metrics.erl

```diff
@@ -288,6 +288,22 @@ register() ->
 		{labels, [task]},
 		{help, "The number of items in the mining server task queue."}
 	]),
+	prometheus_gauge:new([
+		{name, mining_solution_failure},
+		{labels, [reason]},
+		{help, "The number of times we failed to prepare a block from a mining solution."}
+	]),
+	prometheus_gauge:new([
+		{name, mining_solution_success},
+		{help, "The number of times a block was successfully prepared "
+				"from a mining solution. It does not necessarily mean the block "
+				"ended up in the blockchain."}
+	]),
+	prometheus_gauge:new([
```

Comment:
I think we can probably leave this metric out? at least in grafana I would typically just sum the other two metrics to get a graph of the total

Reply:
I can make this edit.

Reply:
https://github.com/ArweaveTeam/arweave/pull/749/commits/317f549b3ae81b9914e687de349a7d9b2973336c